Documentation/UserRoundTrip.rst

```diff
@@ -0,0 +1,345 @@
+
+The issues URL is considered here because it is closely related to the VCS
+repository aspect.
+
+
+.. _user-round-trip-ter-aspect:
+
+TYPO3 Extension Repository
+--------------------------
+
+Guide the user from the TER page to the other aspects by setting these
+configuration values on the
+`extension management page <https://extensions.typo3.org/my-extensions>`__:
+
+*  **Published on Packagist:** check
+*  **Composer name of extension:** `<package-name>`
+*  **Link to issue tracker:** `<vcs-repository-issues>`
+*  **Link to repository:** `<vcs-repository>`
+
+for example for the *Static File Cache* extension:
+
+*  **Published on Packagist:** check
+*  **Composer name of extension:** lochmueller/staticfilecache
+*  **Link to issue tracker:** \https://github.com/lochmueller/staticfilecache/issues
+*  **Link to repository:** \https://github.com/lochmueller/staticfilecache
+
+and find the links to the other aspects presented in the right column of the TER
+page (see `Static File Cache example <https://extensions.typo3.org/extension/staticfilecache>`__).
+
+The documentation URL is automatically resolved when the documentation is
+published on docs.typo3.org. The issues URL is considered here because it is
+closely related to the VCS repository aspect.
+
+
+.. _user-round-trip-project-types:
+
+Project types
+=============
+
+
+.. _user-round-trip-typo3-extension-type:
+
+TYPO3 extension
+---------------
+
+.. uml::
+
+   !include <tupadr3/font-awesome-5/typo3>
+
+   skinparam DefaultTextAlignment center
+   skinparam RectangleBackgroundColor transparent
+   skinparam RectangleBorderColor transparent
+   skinparam UsecaseFontColor<< inactive >> #b9b9b9
+   skinparam UsecaseBorderColor<< inactive >> #b9b9b9
+   skinparam UsecaseBackgroundColor<< inactive >> #FEFEFE
+
+   rectangle "<color:#f49700><$typo3></color>\n TYPO3 extension" as typo3actor
+   usecase "Packagist" as packagist
+   usecase "VCS repository" as vcsRepository
+   usecase "Documentation" as documentation
+   usecase "TYPO3 Extension Repository" as ter
+
+   typo3actor -left-> packagist
+   typo3actor -right-> vcsRepository
+   typo3actor -up-> documentation
+   typo3actor -down-> ter
+
+   packagist <-up-> documentation
+   documentation <-down-> vcsRepository
+   vcsRepository <-down-> ter
+   ter <-up-> packagist
+
+   hide << inactive >> stereotype
+
+Guide the user to all four aspects of the TYPO3 extension, which is usually
+hosted on both TER and Packagist, by configuring:
+
+#. :ref:`user-round-trip-documentation-aspect`
+#. :ref:`user-round-trip-vcs-repository-aspect`
+#. :ref:`user-round-trip-ter-aspect`
+#. :ref:`user-round-trip-packagist-aspect`
+
+For example, look at the *Apache Solr* extension and try navigating all four
+aspects by simply clicking on a redirect link on each page - starting at the
+`Apache Solr TER page <https://extensions.typo3.org/extension/solr>`__.
+
+
+.. _user-round-trip-composer-package-type:
+
+Composer package
```

Comment:
Each extension is - or should be a Composer package. We should clarify this.

Reply:
This does not have to be explained on this page, as there are many pages in the official docs that explain it. On this page the user already knows.